index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Live CSS Editor</title>
+    <!-- Compiled and minified CSS -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"
+    />
+    <!-- Custom css -->
+    <link rel="stylesheet" href="assets/css/style.css" />
+  </head>
+  <body>
+    <nav>
+      <div class="nav-wrapper">
+        <a href="#" class="brand-logo center">Live CSS Editor</a>
+      </div>
+    </nav>
+    <div class="container">
+      <p>
+        Note: Always put <b>!important</b> after css. if you want to change all
+        elements. Example:
+      </p>
+      <code>
+        *{ color: red !important; }
+      </code>
+      <p><i>Why? Because here I use a css framework!</i></p>
+      <div class="row mt">
+        <form class="col s12">
+          <div class="row">
+            <div class="input-field col s12">
+              <textarea id="css-editor" class="materialize-textarea"></textarea>
+              <label for="css-editor">Css Input</label>
+            </div>
+          </div>
+        </form>
```

Comment:
I would've opted out from using a form.. I think it is redundant and just a textarea would suffice.

Reply:
Yeah! You are right. I will fix that :+1: Thanks :pray: